### DISCUSSION TITLE
Icecast 2.4.4 new SSL tls 1.2 error

### DISCUSSION THREAD

**Discussion by caf2022** [savonet/liquidsoap#3736] [General]
Someone know how to fix the problem with icecast 2.4.4 with news ssl with tls 1.2 that won't work? The icecast page show err_ssl_version_or_cipher_mismatch thanks

**Comment by vitoyucepi** [comment]
Hi @caf2022,
Please describe your problem in detail.

**Reply by caf2022** [reply]
After a SSL certificate renew the icecast won't access with https, show message on browser with error err_ssl_version_or_cipher_mismatch and after some search, I think that the problem is the TLS old version of icecast 2.4.4 because letsencripty is generating now the SSL certificates with TLS 1.3 and the last icecast version is 2.4.4 from 2018

**Reply by vitoyucepi** [reply]
I'm not sure if icecast can't use *tls1.3* because it depends on openssl.
Anyway, I can't identify the problem from your description. I need more details.
At least I need your OS version, icecast installation method and configuration.
If you'd like, you can add a link to your icecast.

And finally, this is not an icecast support forum. I can give some advice, but if you think this is definitely a bug, then write a bug report to your icecast vendor.
1. If you are installing from distribution repositories, then post to a distribution bug tracker.
1. If you install it from other packages, then file a bug there.
1. If you are using docker, send a bug report to the image author.
1. If you're building icecast from source, then try reporting to the official icecast git repository.
